### ISSUE TITLE
PDF rendering issue with missing text layer

### ISSUE THREAD

**Issue by jul059** [mozilla/pdf.js#15175]
Attach (recommended) or Link to PDF file here: [file.pdf](https://github.com/mozilla/pdf.js/files/9129339/file.pdf)

Configuration:
- Web browser and its version: Firefox 102.0.1
- Operating system and its version: Windows 7 x64
- PDF.js version: the one included in Firefox 102.0.1
- Is a browser extension: no

Steps to reproduce the problem:
1. Open the pdf in firefox
2. The text layer quickly disappears

What is the expected behavior? (add screenshot): Text layer present.

![image](https://user-images.githubusercontent.com/25236477/179441964-f199a702-0fae-4b0c-9bce-90c2e80b25b7.png)

What went wrong? (add screenshot)

![image](https://user-images.githubusercontent.com/25236477/179441908-c0b46f02-0cbc-405f-8d29-92ee74df5956.png)

**Comment by Snuffleupagus** [comment]
WFM, using PDF.js version `2.15.259` in Firefox Nightly 104.

Note that the *invisible textLayer*, which is what allows you to select/copy text and search in the document, should actually still be present. What more likely happens, in your case, is that the images that make up the document are not rendered as intended.
So, are there any error messages in the console?

**Comment by jul059** [comment]
> WFM, using PDF.js version `2.15.259` in Firefox Nightly 104.
> 
> Note that the _invisible textLayer_, which is what allows you to select/copy text and search in the document, should actually still be present. What more likely happens, in your case, is that the images that make up the document are not rendered as intended. So, are there any error messages in the console?

You are right, the text is still searchable. The browser console displays _'TYPE1STANDARD' is an invalid string label_ when the text image disappears after being displayed briefly.

How do I figure out which version of PDF.js is used by firefox? The issue is also present in Zotero 6.0.9 which uses PDF.js, but it's unclear to me which version.

**Comment by jul059** [comment]
> > The browser console displays _'TYPE1STANDARD' is an invalid string label_ when the text image disappears after being displayed briefly.
> 
> That message should be completely unrelated to this issue. Can you please check the _web_ console instead, opened with `Ctrl + Shift + K`, since I'm not sure if the browser console will contain all possibly relevant messages?
> 
> Also, note that the _image_ is likely never actually becoming visible in your case. What actually happens is that this PDF document first (visibly) renders its text-content using standard PDF text-rendering operators, and that's then being hidden before the _image_ is (supposed to) being rendered. The issue, without being able to reproduce it myself, seems to happen only at that last step.
> 
> > How do I figure out which version of PDF.js is used by firefox?
> 
> It's being printed in the web-console when a PDF document is opened. Also, you may want to test using Nightly: https://nightly.mozilla.org/

web console:
![image](https://user-images.githubusercontent.com/25236477/179538235-ee63632a-7d4c-4b37-aa0f-a51b089c52fd.png)

The last warning appears when the text disappears.

I will try to test nightly later.

**Comment by Snuffleupagus** [comment]
Closing as WFM, since this is a fairly old issue that no one else seems to have been able to reproduce.
Also, note that the images in question are regular JPEG images that aren't even particularly large.
